.gitignore

```diff
@@ -1,6 +1,7 @@
 # Object files
 /cli/Debug
 /cli/Release
+objs/
```

Comment:
I'd prefer to limit the paths that aren't homed and aren't extensions, to avoid possible surprises.  Can we be explicit - `/objs/`, `/7zip/objs/`, etc. as needed?  Though I suppose it only needs `/objs/`?

I'd also kinda prefer obj, but I guess it doesn't matter.  After all, there's no `srcs/` folder.

-[Unknown]

Reply:
I changed it to `obj` and you are right, I needed to add `/7zip/obj` as well. While the main Makefile will create all the obj files in the same directory `/obj/` and `/obj/7zip/` the user can still go into the `7zip` directory and have it create the `obj` directory there instead.